### DISCUSSION TITLE
Could it be possible to had an auto player in Langit since Roman doesn't work anymore

### DISCUSSION THREAD

**Discussion by Sullywn** [thatskymod/Sky-CotL-Scripts#152] [Ideas]
I wanted to ask if it's possible to add an auto music player in Langit, because Roman doesn't work 😓

**Comment by thatskymod** [comment]
Adding an automatic music player to Dukun Langit would technically be possible given that both mods are built on the same architecture. However, I can't guarantee whether this is on their development roadmap. I suggest bringing it up in the [Langit Discussion Board](https://t.me/dukunlangit).

The thatskymod GitHub is simply a collection of resources available to us, the end users—we can't make decisions or implement changes on our own. Threads listed here are unlikely to be seen by original developers.
